editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -457,7 +457,7 @@ function printUtopiaJSXComponent(
   detailOfExports: ExportsDetail,
 ): TS.Node {
   const asJSX = jsxElementToExpression(element.rootElement, imports, printOptions.stripUIDs)
-  if (TS.isJsxElement(asJSX) || TS.isJsxSelfClosingElement(asJSX)) {
+  if (TS.isJsxElement(asJSX) || TS.isJsxSelfClosingElement(asJSX) || TS.isJsxFragment(asJSX)) {
```

Comment:
Is this a bug fix?

Reply:
It is, mentioned this in the description above.